### PR TITLE
Smart contract

### PR DESCRIPTION
**Pull request by ayushvasu**

### CODE REVIEW

**Comment by karthikprasad13** [comment]
Code is not in proper format. We should format code before merging this request. Also, we need to update [CONTRIBUTING.md](https://github.com/Impetus/eth-jdbc-connector/blob/master/CONTRIBUTING.md) with a link to downloadable formatter.

-Karthik

**Comment by ayushvasu** [comment]
I have formatted the code with Kundera like code formatter

File: eth-jdbc-examples/src/main/java/com/impetus/blkchn/eth/SmartContractDeploy.java [hunk 1]
```diff
@@ -13,31 +28,29 @@
...
+			boolean retBool = stmt.execute(query);
+			if (retBool) {
+				ResultSet ret = stmt.getResultSet();
+				ret.next();
+				String return_value = (String) ret.getObject(1);
+				LOGGER.info("Smart contract has deployed at address :: " + return_value);
+			}
+			LOGGER.info("done");
+		} catch (Exception e) {
+			LOGGER.info(e.getMessage());
```
Comment (karthikprasad13) [inline eth-jdbc-examples/src/main/java/com/impetus/blkchn/eth/SmartContractDeploy.java]: Use `LOGGER.error()` in catch block while printing error message

**Comment by karthikprasad13** [comment]
@ayushvasu 

I think `SmartContractDeploy.java`, `SmartContractDeployAsync.java` and `SmartContractFunction.java` are more like test classes and should not be in `src/main/java`. We should make them as test cases and move to `src/test/java`. Discuss with @ashishkshukla

**Comment by ayushvasu** [comment]
@karthikprasad13  
Integration test cases are there for these functionality, I will discuss with @ashishkshukla,if we have to remove it from example module and also have to add wiki for this feature

**Comment by ashishkshukla** [comment]
@karthikprasad13 These classes are created in examples module not in the driver one. So I think we can keep it there if someone wants to run the examples. 
Anyways I'll take a closer look and will update it accordingly

**Comment by ashishkshukla** [comment]
@ayushvasu add integration tests for the condition -if deployed contract address in not valid in the given query.
And also separate out smart contract specific work in different package in examples module.

**Pull request merged**